_remake/main.js

```diff
@@ -175,17 +176,13 @@ if (RemakeStore.isMultiTenant()) {
     }
   }
   
-  // open MySQL connection
-  global.connection = mysql.createConnection({
+  global.database = mysql.createPool({
```

Comment:
Are we creating a pool of databases instead of just one now? What was the underlying issue? Too many connections to the single database?